borgmatic/hooks/mysql.py

```diff
@@ -79,8 +79,9 @@ def execute_dump_command(
         )
         return None
 
+    mysql_dump_command = database.get('mysql_dump_command') or 'mysqldump'
```

Comment:
To prevent shell injections attacks, this (and all custom commands in `mysql.py`) need to be escaped. See the `postgresql.py` hook (like on like 77) for an example.